test/test.js

```diff
@@ -24,6 +24,28 @@ describe('install', function() {
     assert(context.$duv && context.$dum && context.$dus);
   });
 
+  it('should install all the human readable functions on the global NS', function() {
+    this.run();
+    var context = this.context;
+    assert(
+      context.$duEvent &&
+      context.$duEventLogger &&
+      context.$duEventRemove &&
+      context.$duWrapMethod &&
+      context.$duWrapMethodLogger &&
+      context.$duWrapMethodRemove &&
+	  context.$duGetter &&
```

Comment:
Damn! I'm usually so particular about that. :( I guess it's because I'm set up for tabs not spaces. :(

My left arm for a source control system that is SOURCE control, not the *representation* of it!